src/utils/logLibVersions.ts

```diff
@@ -73,8 +73,9 @@ export default async () => {
         logVersion(versions, 'nrfjprog_dll', 'nrfjprog dll');
         logVersion(versions, 'jlink_dll', 'JLink');
     } catch (error) {
-        logger.error(
-            `Failed to get the library versions: ${error.message || error}`
-        );
+        const message = error instanceof Error ? error.message : error;
+        if (error instanceof Error) {
+            logger.error(`Failed to get the library versions: ${message}`);
+        }
```

Comment:
Why do we only log if it's an instance of `Error`? We still have the message right?

Reply:
Typescript won't let us access the message property before checking the type of the error. In theory "anything" can be the type of the exception caught.